### PR TITLE
Add Markdeep for LaTex support in markdown

### PR DESCRIPTION
**Pull request by DragoonX6**
This adds:
 * Markdeep
* A template and short instructions for using Markdeep
* A GSC search bear

### CODE REVIEW

**Review by nickclark2016 — APPROVED** [review]
Good to go.

File: LaTexMD/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Using LaTex in markdown
+
+Since GitHub markdown doesn't natively support LaTex, we use markdeep.
```
Comment (deccer) [inline LaTexMD/index.md:3]: [markdeep](https://casual-effects.com/markdeep/)

Comment (deccer) [reply LaTexMD/index.md:3]: ah great, review sucks here, i did only select markdeep in `we use markdeep`

File: LaTexMD/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Since GitHub markdown doesn't natively support LaTex, we use markdeep.  
+Specifically, a slightly modified version of markdeep to look like GitHub markdown.
+
+Using markdeep is relatively straight forward. You write your markdown as you normally would, and use the special markdeep syntax for its extra features (the syntax is explained on [their website](https://casual-effects.com/markdeep/)).  
+Besides that you add the script snippets at the bottom of your markdown file, and save it with the extension `.md.html`.  
+The script snippets can be found in [template.md.html](template.md.html).
+
+# Requirements
+
+1. You are not allowed to use a different style sheet, we want all pages to look uniform across the website.
```
Comment (deccer) [inline LaTexMD/index.md:12]: Hmm... maybe what about a slightly different wording. instead of `you are not allowed to...` write `we encourage you to use style x`

**Review by deccer — APPROVED** [review]
Approving with suggestions :)

File: LaTexMD/index.md (hunk 2 above)
Comment (nickclark2016) [reply LaTexMD/index.md:12]: This was actually one of my requirements for the website.  We are aiming for uniformity, which makes it easier to read.

Comment (DragoonX6) [reply LaTexMD/index.md:12]: How is encouraging a better word when the reality is that we're rejecting PRs that are in a different style?

File: LaTexMD/index.md (hunk 1 above)
Comment (DragoonX6) [reply LaTexMD/index.md:3]: I'm not sure what you're saying here. Can you rephrase?

**Comment by nickclark2016** [comment]
Something I just realized.  Is it possible to get markdeep from a CDN rather than hosting it in our repository?

**Comment by DragoonX6** [comment]
We can do that, but then everything will basically look like `index.html`, and that's much more effort to use than the template we have now. Besides, I patched Markdeep to fix a bug with rendering HTML tags.

**Comment by DragoonX6** [comment]
Ok this is weird, GitHub didn't preserve the order of my commits, and I have no idea why.

I think it's alright, but maybe do a manual merge to be sure.

File: LaTexMD/index.md (hunk 2 above)
Comment (deccer) [reply LaTexMD/index.md:12]: I get scolded for being real on our server, but here its fine? I don't mind being strict. I was also just saying

File: LaTexMD/index.md (hunk 1 above)
Comment (deccer) [reply LaTexMD/index.md:3]: I wanted the word `markdeep` itself to be a link already

Comment (DragoonX6) [reply LaTexMD/index.md:3]: Oh, sure, I'll fix that in a bit.
